modules/nw-sctp-about.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * networking/using-sctp.adoc
+
+[id="nw-sctp-about_{context}"]
+= Support for Stream Control Transmission Protocol (SCTP) on {product-title}
+
+If your cluster is installed on bare metal infrastructure, you can enable SCTP on the hosts in the cluster. On {op-system-first} the SCTP module is blacklisted by default.
```

Comment:
Not sure to just menetion bare metal infrastructure here, because QE can run SCTP testing in AWS